api/dbadapters/redshift.ts

```diff
@@ -74,7 +74,7 @@ export class RedshiftDbAdapter implements IDbAdapter {
       if (options.includeQueryInError) {
         throw new Error(`Error encountered while running "${statement}": ${e.message}`);
       }
-      throw new ErrorWithCause(`Error executing Redshift query: ${e.message}`, e);
```

Comment:
we shouldn't remove this, we added it for a reason (check blame if you like, but TL; DR: we wanted to clarify whether errors were in the warehouse, or dataform).

instead we should pass a constructor parameter to the DbAdapter that tells it which context it's being used for, and use that.

Reply:
Ah I see, have done this now